rosidl_typesupport_introspection_c/resource/srv__type_support.c.em

```diff
@@ -61,7 +61,7 @@ function_prefix = '__'.join(include_parts) + '__rosidl_typesupport_introspection
 static rosidl_typesupport_introspection_c__ServiceMembers @(function_prefix)__@(service.namespaced_type.name)_service_members = {
   "@('__'.join([package_name] + list(interface_path.parents[0].parts)))",  // service namespace
   "@(service.namespaced_type.name)",  // service name
-  // these two fields are initialized below on the first access
+  // these three fields are initialized below on the first access
```

Comment:
Maybe use 'the following fields' instead.

Reply:
Yeah, agreed.  I think this should be:

```suggestion
  // the following fields are initialized below on first access
```